lib/cdo/i18n_string_url_tracker.rb

```diff
@@ -26,6 +28,29 @@ def log(string_key, url, source)
 
   private
 
+  # Determines if this URL should be tracked. We want to filter URLs so we don't waste resources recording data we are
+  # not interested in.
+  def allowed(url)
+    return false unless url
+    parsed_url = URI(url)
+
+    # Include any non-studio.code.org URLs e.g. code.org, hourofcode.com, etc
+    return true unless parsed_url.host&.match(/.*studio\.code\.org.*/)
+
+    # Allow Applab/Weblab/Gamelab/Project based URLs
+    # Example: https://studio.code.org/projects/flappy/zjiufOp0h-9GS-DywevS0d3tKJyjdbQZZqZVaiuAjiU/view
+    return true if parsed_url.path&.match(/\/projects\/.*/)
```

Comment:
Do we have any kind of contract/guarantee that these URL paths will always have the same prefixes?